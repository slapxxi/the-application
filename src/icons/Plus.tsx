import * as React from 'react';

interface Props {
  size?: number;
}

function Plus({ size = 16 }: Props) {
  return (
    <svg width={size} height={size} viewBox="0 0 100 100">
      <title>Plus Icon</title>
      <polygon points="100 38.19 61.81 38.19 61.81 0 38.19 0 38.19 38.19 0 38.19 0 61.81 38.19 61.81 38.19 100 61.81 100 61.81 61.81 100 61.81 100 38.19" />
    </svg>
  );
}

export default Plus;
