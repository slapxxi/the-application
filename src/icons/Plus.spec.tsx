import React from 'react';
import { render } from 'react-testing-library';
import Plus from './Plus';

it('renders', () => {
  const { container } = render(<Plus />);
  expect(container.firstChild).toMatchSnapshot();
});

it('accepts "size" prop', () => {
  const { container } = render(<Plus size={32} />);
  expect(container.firstChild).toMatchSnapshot();
});
