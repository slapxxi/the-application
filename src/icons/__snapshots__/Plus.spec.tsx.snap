// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`accepts "size" prop 1`] = `
<svg
  height="32"
  viewBox="0 0 100 100"
  width="32"
>
  <title>
    Plus Icon
  </title>
  <polygon
    points="100 38.19 61.81 38.19 61.81 0 38.19 0 38.19 38.19 0 38.19 0 61.81 38.19 61.81 38.19 100 61.81 100 61.81 61.81 100 61.81 100 38.19"
  />
</svg>
`;

exports[`renders 1`] = `
<svg
  height="16"
  viewBox="0 0 100 100"
  width="16"
>
  <title>
    Plus Icon
  </title>
  <polygon
    points="100 38.19 61.81 38.19 61.81 0 38.19 0 38.19 38.19 0 38.19 0 61.81 38.19 61.81 38.19 100 61.81 100 61.81 61.81 100 61.81 100 38.19"
  />
</svg>
`;
